RecycleView/app/src/main/java/net/boondockradio/recycleview/ItemViewHolder.java

```diff
@@ -0,0 +1,31 @@
+package net.boondockradio.recycleview;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class ItemViewHolder extends RecyclerView.ViewHolder {
+
+    public TextView textView;
+
+    public ItemViewHolder(View v) {
+        super(v);
+        textView = (TextView) v.findViewById(R.id.text_raw_item);
+        textView.setOnClickListener(new View.OnClickListener() {
```

Comment:
its more flexiable to have `setOnClickListener` in `onBindViewHolder`, we can access entire dataset

for example if we wanted to make a `Toast` that says
`I love Florida, not Georgia`

we could write
`String message = context.getString(R.string.states_lover, states.get(position) , states.get(position) + 1);`